cmd/client/main.go

```diff
@@ -1,10 +1,16 @@
 package main
 
 import (
-	"github.com/ASeegull/SmartFridge/examplePackage"
 	"fmt"
+	"log"
+	"net/http"
+
+	s "github.com/ASeegull/SmartFridge/staticServer"
```

Comment:
Try to avoid not human readable names like `s`

Reply:
Changed) thought it was ok, 'cause there were just few lines of code. Can I use short names inside functions?